djangorarapi/views/comments.py

```diff
@@ -16,6 +16,30 @@
 
 class Comments(ViewSet):
 
+    def update(self, request, pk=None):
```

Comment:
May want to rename PR to reflect update versus delete before merging to main.

Reply:
Will do!